src/event/mod.rs

```diff
@@ -160,6 +156,15 @@ impl<K: Into<Atom>, V: Into<Value>> FromIterator<(K, V)> for LogEvent {
     }
 }
 
+impl Serialize for LogEvent {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        serializer.collect_map(self.fields.clone())
```

Comment:
can we avoid this clone here? 

https://docs.rs/serde/1.0.104/serde/trait.Serializer.html#method.collect_map

looks like the example here uses a btreeset.

Reply:
`self.fields.clone()` is replaced by `self.fields.iter()` now.